Update README.md

anajsana opened the pull request:
- Included "What is a Steering Committee" using the wording from the CHAOSS governance as a reference in their [governing board section](https://chaoss.community/governance/)
- Included non-profit 501 (c)(6) entity term under "Why Linux Foundation" question. Additionally, I improved the wording with concrete examples to clearly state that Developer Relations Foundations follows a community-driven, LF unfunded model, and is NOT an LF-funded model.
- Included new question for those new to OS projects called "What is a Charter" with the charter link and be explicit that charters are living documents and can have multiple versions in the future, as the project evolves

See preview [here](https://github.com/DevRel-Foundation/.github/blob/04fc0f6548b12c48e5bdadb7da7181222159d9d9/profile/README.md)

anajsana committed:
Update README.md

include "what is a Steering Committee" and volunteer SC term

leggetter commented on profile/README.md:
```diff
@@ -48,12 +48,19 @@ If you have questions that are not answered here, please get in touch via the [D
+  <summary>
+    <strong>Who is the current Steering Committee?</strong>
+  </summary><br />
+  There is currently a temporary Volunteer Steering Committee. We, the volunteer Steering Committe participants, are composed of tenured Developer Relations professionals, who initiated this “intent to form foundation” built through the LF’s guidance and principles. Once the   charter and mission are in place, a new leadership will be    established. This process will be via elections at a future ...
```
```suggestion
  There is currently a temporary Volunteer Steering Committee. We, the volunteer Steering Committee participants, are composed of tenured Developer Relations professionals who initiated this “intent to form foundation” built through the LF’s guidance and principles. A new leadership will be established once the charter and mission are in place. This process will be via elections at a future date to be determined.
```

leggetter commented on profile/README.md:
```diff
@@ -48,12 +48,19 @@ If you have questions that are not answered here, please get in touch via the [D
+  There is currently a temporary Volunteer Steering Committee. We, the volunteer Steering Committe participants, are composed of tenured Developer Relations professionals, who initiated this “intent to form foundation” built through the LF’s guidance and principles. Once the   charter and mission are in place, a new leadership will be    established. This process will be via elections at a future ...
 
-  While our profession is relatively small, it is big enough that there are many really good people who can do this work. And we hope they want to! We had the initiative and time to do this, and so we feel it’s reasonable for us, just like anyone else, to take the first steps forward on building something
+  While our profession is relatively small, it is big enough that there are many really good people who can do this work. And we hope they want to! We had the initiative and time to do this, and so we feel it’s reasonable for us, just like anyone else, to take the first steps       forward on building something
```
```suggestion
  While our profession is relatively small, it is big enough that there are many really good people who can do this work. And we hope they want to! We had the initiative and time to do this, and so we feel it’s reasonable for us, just like anyone else, to take the first steps forward on building something
```

leggetter reviewed (changes requested):
A few small change requests

anajsana committed:
Update profile/README.md

Co-authored-by: Phil Leggetter <phil@leggetter.co.uk>

anajsana committed:
Update profile/README.md

Co-authored-by: Phil Leggetter <phil@leggetter.co.uk>

anajsana committed:
Update README.md

Within teh FAQ section: adding a few edits on the "what is an SC" using the wording from the CHAOSS governance as a reference in their [governing board section](https://chaoss.community/governance/)

Also added a new question for those new to OS projects called "What is a charter" and be explicit that charters are living documents and can have multiple versions in the future, as the project evolves

anajsana commented:
> A few small change requests

thanks! I've just committed your changes and made a few edits re FAQ section:

- Added a few edits on the "what is an SC" using the wording from the CHAOSS governance as a reference in their [governing board section](https://chaoss.community/governance/)

- Added a new question for those new to OS projects called "What is a charter" and be explicit that charters are living documents and can have multiple versions in the future, as the project evolves

anajsana committed:
Update README.md

Include documentation in teh FAQ about Developer Relations Foundation being part of the LF project series, a non-profit 501(c)(6).

jcleblanc reviewed (changes requested):
The content lgtm with the exception of some small tweaks to line 66: 
"There is currently a temporary Steering Committee...." as the "temporary" was raised as a confusing point in our last meeting. 

I'd recommend dropped that and instead just stating:
"The current Steering Committee participants are composed of tenured Developer Relations professionals who initiated this “intent to form foundation” built through the LF’s guidance and principles. A new leadership will be established once the charter and mission are in place. This process will be via elections at a future date to be determined."

anajsana commented on profile/README.md:
```diff
@@ -31,27 +31,39 @@ If you have questions that are not answered here, please get in touch via the [D
   </summary><br />
-  
-  This first, non-official Interim SC, composed of tenured Developer Relations professionals, has initiated this “intent to form foundation” built through the LF’s guidance and principles. Once the charter and mission are in place, a new Leadership council will be established. This process will be via elections at a future date to be determined.
+  There is currently a temporary Steering Committee. We, the Steering Committee participants, are composed of tenured Developer Relations professionals who initiated this “intent to form foundation” built through the LF’s guidance and principles. A new leadership will be established once the charter and mission are in place. This process will be via elections at a future date to be determined.
```
```suggestion
 The current Steering Committee participants are composed of tenured Developer Relations professionals who initiated this “intent to form foundation” built through the LF’s guidance and principles. A new leadership will be established once the charter and mission are in place. This process will be via elections at a future date to be determined.
```

anajsana committed:
Update README.md

anajsana commented:
> The content lgtm with the exception of some small tweaks to line 66: 
> 
> "There is currently a temporary Steering Committee...." as the "temporary" was raised as a confusing point in our last meeting. 
> 
> 
> 
> I'd recommend dropped that and instead just stating:
> 
> "The current Steering Committee participants are composed of tenured Developer Relations professionals who initiated this “intent to form foundation” built through the LF’s guidance and principles. A new leadership will be established once the charter and mission are in place. This process will be via elections at a future date to be determined."
> 
> 
> 
> 

Thanks for your review ! I've made the requested changes in line 66 based on your comment 🙂

jcleblanc reviewed (approved):
lgtm; thanks!

Cruzn65 commented:
Thank you. Please remove Temporary or Interim from Github as I think I saw it elsewhere. CheerS!
